app/models/payout_report.rb

```diff
@@ -12,15 +14,36 @@ def encryption_key
     Rails.application.secrets[:attr_encrypted_key]
   end
 
-  def self.total_amount
-    PayoutReport.sum { |payout_report| payout_report.amount.to_i } / 1E18
+  def amount
+    PotentialPayment.where(payout_report_id: id).sum { |potential_payment| potential_payment.amount.to_i }
```

Comment:
`potention_payments.sum { |potential_payment| potential_payment.amount.to_i }` since you already have a `has_many` relationship.